src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -428,5 +429,46 @@ private enum TimeSuffix
 			Ms,
 			S
 		}
+
+
+		protected string ParseCaptureBody(ConfigurationKeyValue kv)
+		{
+			var captureBodyInConfig = kv.Value;
+			if (string.IsNullOrEmpty(captureBodyInConfig))
+			{
+				return DefaultValues.CaptureBody;
+			}
+
+			if (!SupportedValues.CaptureBodySupportedValues.Contains(captureBodyInConfig.ToLowerInvariant()))
+			{
+				Logger?.Error()
+				?.Log("The CaptureBody value that was provided ('{DefaultServiceName}') in the configuration is not allowed. request body will not be captured." +
+					"The supported values are : ",
+					captureBodyInConfig.ToLowerInvariant(), string.Join(", ", SupportedValues.CaptureBodySupportedValues));
+				return DefaultValues.CaptureBody;
+			}
+			return captureBodyInConfig.ToLowerInvariant();
+		}
+
+		protected List<string> ParseCaptureBodyContentTypes(ConfigurationKeyValue kv, string captureBody)
+		{
+			var captureBodyContentTypesInConfig = kv.Value;
+
+			//CaptureBodyContentTypes and CaptureBody are linked. Verify that in case CaptureBody is ON - then CaptureBodyContentTypes is not empty
+			if (string.IsNullOrEmpty(captureBodyContentTypesInConfig) && captureBody != SupportedValues.CaptureBodyOff)
+			{
+				Logger?.Error()?.Log("Capture_Body is on but no content types are configured. Request bodies will not be captured.");
+				return new List<string>();
+			}
+
+			var captureBodyContentTypes = new List<string>();
+			if (captureBodyContentTypesInConfig != null)
+			{
```

Comment:
Same as above: `{` and `}` can be removed for `if` bodies with a single line. This is defined in our `.editorconfig`

<img width="1103" alt="image" src="https://user-images.githubusercontent.com/1091853/62858567-fdf44200-bcfa-11e9-96ed-2370097648ab.png">

Reply:
Done.